conda.recipe/meta.yaml

```diff
@@ -10,6 +10,7 @@ requirements:
     - setuptools
   run:
     - anaconda-client
+    - nb_config_manager
     - nbsetuptools
     - notebook
```

Comment:
You don't need the notebook dependency here because it is handled by the recipe in `nb_config_manager`...